ci-scripts/test_agent/main.py

```diff
@@ -64,8 +65,10 @@ def main():
                     {"state": tester.get_state(), "current_workload": None}
                 )
 
+        time.sleep(1)
+        db_client.user = db_client.auth.refresh(db_client.user["refreshToken"])
```

Comment:
Tokens are good for 1 hr. Refreshing every 15 seconds is too much.
Lets make it refresh every 15-30 min

Reply:
👍